### PR TITLE
Fix stats mk

### PR DESCRIPTION
**Pull request by Fedja**

### CODE REVIEW

File: data/C3_PROSTATE_EXALLC_withp_alleles_eur.csv [hunk 1]
```diff
@@ -0,0 +1,146 @@
+trait	#chrom	pos	ref	alt	beta	se	pval	study_doi
+PRCA	1	10496039	G	A	0.039220713		5.90E-07	https://doi.org/10.1038/s41588-018-0142-8
+PRCA	1	150685810	A	G	0.067658648		9.90E-12	https://doi.org/10.1038/s41588-018-0142-9
```
Comment (sinarueeger) [inline data/C3_PROSTATE_EXALLC_withp_alleles_eur.csv:3]: study doi is not constant.

File: data/melanoma.csv [hunk 2]
```diff
@@ -0,0 +1,23 @@
+trait	#chrom	pos	ref	alt	beta	se	pval	study_doi
+MELANOMA	1	150887995	C	T	0.139761942		9.00E-11	10.18632/oncotarget.15250
```
Comment (sinarueeger) [inline data/melanoma.csv:2]: study doi not constant

File: data/basal_cell_carcinoma.csv [hunk 3]
```diff
@@ -0,0 +1,35 @@
+trait	#chrom	pos	ref	alt	beta	se	pval	study_doi
+basal cell carcinoma	6	396321	C	T	0.392042088		2.4E-152	https://doi.org/10.1038/ncomms12510
+basal cell carcinoma	16	89919709	C	T	0.336472237		2.5E-63	https://doi.org/10.1038/ncomms12511
```
Comment (sinarueeger) [inline data/basal_cell_carcinoma.csv:3]: study doi not constant

File: match_file.csv [hunk 4]
```diff
@@ -1,26 +1,42 @@
...
 J10_ASTHMA.csv	J10_ASTHMA
 J10_COPD.csv	J10_COPD
 L12_ATOPIC.csv	L12_ATOPIC
 L12_PSORIASIS.csv	L12_PSORIASIS
 M13_ANKYLOSPON.csv	M13_ANKYLOSPON
+M13_ANKYLOSPON.csv	M13_ANKYLOSPON_STRICT
 T1D.csv	T1D
 T2D.csv	T2D
 RHEUMA_SEROPOS.csv	RHEUMA_SEROPOS
+melanoma.csv	C3_SKIN_EXALLC
```
Comment (sinarueeger) [inline match_file.csv:34]: For skin cancer there are now 3 files: 
- C3_SKIN_EXALLC.csv
- melanoma.csv
- basal_cell_carcinoma.csv
What is the purpose of having both melanoma.csv and C3_SKIN_EXALLC.csv? 
They are covering the same phenotype and one could be removed.

File: match_file.csv [hunk 5]
```diff
@@ -1,26 +1,42 @@
 EXT	FG
 C3_BREAST_EXALLC.csv	C3_BREAST_EXALLC
 C3_PROSTATE_EXALLC.csv	C3_PROSTATE_EXALLC
+C3_PROSTATE_EXALLC_withp_alleles_eur.csv	C3_PROSTATE_EXALLC
```
Comment (sinarueeger) [inline match_file.csv:4]: why not adding these variants to C3_PROSTATE_EXALLC.csv? is it bc C3_PROSTATE_EXALLC.csv are review results?

File: data/basal_cell_carcinoma.csv (hunk 3 above)
Comment (sinarueeger) [reply data/basal_cell_carcinoma.csv]: fixed it

File: data/melanoma.csv (hunk 2 above)
Comment (sinarueeger) [reply data/melanoma.csv]: fixed

File: data/C3_PROSTATE_EXALLC_withp_alleles_eur.csv (hunk 1 above)
Comment (sinarueeger) [reply data/C3_PROSTATE_EXALLC_withp_alleles_eur.csv]: fixed

File: match_file.csv (hunk 4 above)
Comment (Fedja) [reply match_file.csv:34]: We don't have C3_SKIN_EXALLC.csv
We just compare that scan of ours to melanoma.csv and basal_cell_carcinoma.csv

File: match_file.csv (hunk 5 above)
Comment (Fedja) [reply match_file.csv:4]: yea just left these in so I could see how the results changed.  In the long term they are not needed separately so the other can be removed

File: match_file.csv (hunk 4 above)
Comment (Fedja) [reply match_file.csv:34]: So C3_SKIN_EXALLC.csv has everything combined and then we have subtypes. Just for comparison purposes as our phenotype has mix of both

Comment (sinarueeger) [reply match_file.csv:34]: Ok, so we keep all 3 files and C3_SKIN_EXALLC.csv contains melanoma and basal_cell_carcinoma.

File: match_file.csv (hunk 5 above)
Comment (sinarueeger) [reply match_file.csv:4]: Same file is ok then.

File: match_file.csv [hunk 6]
```diff
@@ -1,26 +1,42 @@
...
+pd_no_proxy_2017.csv	PDSTRICT
+pd_no_proxy_2017.csv	PD2ND_EXMORE
 H7_AMD.csv	H7_AMD
 I9_AF.csv	I9_AF
 I9_HYPERTENSION.csv	I9_HYPERTENSION
 IBD_CD.csv	K11_IBD_STRICT
 IBD_IBD.csv	K11_IBD_STRICT
 IBD_UC.csv	K11_IBD_STRICT
 IBD_UC.csv	K11_ULCER
+IBD_UC.csv	K11_UC_STRICT0
```
Comment (sinarueeger) [inline match_file.csv:23]: Remark for Sina: check if `K11_UC_STRICT0` real endpoint

Comment (sinarueeger) [reply match_file.csv:23]: isnt, just typo

**Pull request closed without merging**